### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Olawumi11**
## Task: `Develop a Contact Form Page using HTML`
- `Create a contact form page`

### CODE REVIEW

**Comment by adaugochi** [comment]
Hello. 
Your contact form should be in your `html-forms` folder and not `html` folder. So, it is required of you to create a new folder called `html-forms` inside your named folder

File: submissions/famuyiwaOlawumi/htmltask/index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/famuyiwaOlawumi/htmltask/index.html:1]: Did you decide to rework your product page again

**Comment by Olawumi11** [comment]
Your review on folder name is clearly different from what instructions 6 of the task [7] said ma.

**Comment by adaugochi** [comment]
ok. I will advise you don't work on two tasks at the same time. get one task merged first before you proceed to another. Cos, now task 8 contain what is on task 7 already

File: submissions/famuyiwaOlawumi/html-forms/htmlform-7.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (adaugochi) [inline submissions/famuyiwaOlawumi/html-forms/htmlform-7.html:3]: rename this file to contact.html

File: submissions/famuyiwaOlawumi/htmltask/index.html (hunk 1 above)
Comment (adaugochi) [reply submissions/famuyiwaOlawumi/htmltask/index.html:1]: You already have an `index.html` file in HTML folder. Is there a reason why you are creating this again?

Comment (ToluAkin) [reply submissions/famuyiwaOlawumi/htmltask/index.html:1]: @Olawumi11 Also, remove the images associated with this file after deleting this page

File: submissions/famuyiwaOlawumi/html-forms/contact.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>contact us</title>
+</head>
+<body>
+    <h1>Contact Form</h1>
+<form action="#" method="post" enctype="multipart/form-data">
```
Comment (ToluAkin) [inline submissions/famuyiwaOlawumi/html-forms/contact.html:9]: You need to remove attributes `action`, `method` and `enctype`. They are not needed yet. @Olawumi11

**Pull request merged**